CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
```

Comment:
in commit message: maybe call the scope `xAPI` instead of `LRS` so that all subjects related to xAPI fall in this scope?

Reply:
:ok_hand: